lib/DDG/Spice/GoWatchIt.pm

```diff
@@ -31,7 +31,8 @@ spice wrap_jsonp_callback => 1;
 handle remainder_lc => sub {
 
   return unless $_; # Guard against "no answer"
-  return if ($req->query_lc ~~ @stopwords); # Guard against stop words "apple watch"
+  return if grep {$req->query_lc eq $_} @stopwords;
```

Comment:
@moollaza can you explain a little how the grep is working? I thought the block was something that we searched for in the array(`@stopwords` in this case). But the block is a conditional in this case?